network-api/networkapi/wagtailpages/pagemodels/customblocks/listing.py

```diff
@@ -31,6 +33,8 @@ def clean(self, value):
 
 
 class ListingBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(help_text="Heading for the block.", required=False)
+
```

Comment:
I wasn't sure whether to add a specific heading here or allow publishers to use another block for the heading so I made this not required. I'm open to removing it though if there is a stronger opinion.